config/config/sc-config.yaml

```diff
@@ -1096,6 +1096,7 @@ networkPolicies:
         - set-me-if-enabled
       ports:
         - set-me-if-enabled
+    ingressUsingHostNetwork: false
```

Comment:
```suggestion
```

Reply:
Harbor-np is still using this.

Reply:
We have this variable in common config.

Reply:
As @Pavan-Gunda mentioned, this is available in common config. That means it can be removed here, right?